custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -448,8 +460,9 @@ def _prepare_docker_command(self, options, run_mode):
         docker_cmd = "docker run --rm --interactive  --user $(id -u):$(id -g) "
         docker_cmd_args = " -v {}:{}".format(options.code_dir, in_docker_model)
 
-        in_docker_cmd_list = sys.argv
+        in_docker_cmd_list = raw_arguments or sys.argv
```

Comment:
Take sys.argv from here to the upper level ...

Reply:
I'm sending in raw_arguments = None by default so that wouldn't work.

Reply:
exactly set `raw_arguments = sys.argv` for default.
Anyhow, finish with docker implementation before